CHANGELOG.md

```diff
@@ -18,6 +19,29 @@
 
 All notable changes to this project will be documented in this file.
 
+## 0.7.2
+
+* ### Merged Pull Requests
+    * [Query: Closed PRs with the label: interpreter_implementation](https://github.com/ni/nidaqmx-python/issues?q=label%3Ainterpreter_implementation+is%3Aclosed)
+    * [Query: Closed PRs with the label: library_interpreter](https://github.com/ni/nidaqmx-python/issues?q=label%3Alibrary_interpreter+is%3Aclosed)
+    * [Query: Closed PRs with the label: grpc_interpreter](https://github.com/ni/nidaqmx-python/issues?q=label%3Alibrary_interpreter+is%3Aclosed)
+    * [Query: Closed PRs with the label: test_improvements to increase the test coverage](https://github.com/ni/nidaqmx-python/issues?q=label%3Atest_improvements+is%3Aclosed)
+    * [Query: Closed PRs with the label: interpreter_fixes to fix all the issues in the interpreters](https://github.com/ni/nidaqmx-python/issues?q=label%3Ainterpreter_fixes+is%3Aclosed)
+    * [Query: Closed PRs with the label: interpreter_testcase_update to make the existing test cases run with both the interpreter](https://github.com/ni/nidaqmx-python/issues?q=label%3Ainterpreter_testcase_updates+is%3Aclosed)
+    * [Query: Closed PRs with the label: event_handling that takes care of the event handling in the interpreters](https://github.com/ni/nidaqmx-python/issues?q=label%3Aevent_handling+is%3Aclosed)
+* ### Resolved Issues
+    * ...
+* ### Major Changes
+    * Introduced library and gRPC intepreters to communicate with the device, either directly or through gRPC respectively.
```

Comment:
Use user-facing terminology. "interpreter" is an internal implementation detail.